pyproject.toml

```
@@ -11,7 +11,7 @@ dependencies = [
     "requests>=2.27.0",
     "requests-toolbelt>=1.0.0",
     "macaroonbakery>=1.3.0,!=1.3.3",
-    "pydantic>=1.10,<2.0",
+    "pydantic~=2.0",
```

```suggestion
    "pydantic~=2.8",
```
As a precaution I'd suggest to use latest release as a accepted minimum just in case